test/e2e/run_e2e.sh

```diff
@@ -44,9 +44,11 @@ CDP_TARGET_REPOSITORY="${CDP_TARGET_REPOSITORY:-"github.com/zalando-incubator/ku
 CDP_TARGET_COMMIT_ID="${CDP_TARGET_COMMIT_ID:-"dev"}"
 CDP_HEAD_COMMIT_ID="${CDP_HEAD_COMMIT_ID:-"$(git describe --tags --always)"}"
 RESULT_BUCKET="${RESULT_BUCKET:-""}"
+CLUSTER_SUFFIX="${CLUSTER_SUFFIX:-""}"
```

Comment:
Why are we adding this `CLUSTER_SUFFIX`? If it's the part of the multi-cluster setup suggested, I'd suggest we drop it.

Reply:
`CLUSTER_SUFFIX` was used to make the LOCAL_ID and ALIAS unique for the 2nd cluster. Will cleanup